Modules/Customer/Http/Controllers/CustomerController.php

```diff
@@ -121,26 +121,25 @@ public function update(UpdateCustomerRequest $request): JsonResponse
     }
 
     /**
-     * Remove the specified campaign from storage
+     * Remove the customers from storage
      *
-     * @param string $customerKey
+     * @param Request $request
      * @return JsonResponse
      */
-    public function destroy(string $customerKey): JsonResponse
+    public function destroy(Request $request): JsonResponse
     {
         $user = auth()->user();
-        $customer = Customer::where('customer_key', $customerKey)->first();
 
-        // return error if user not created the campaign
-        if ($user->cannot('delete', $customer)) {
+        // return error if user is not a brand
+        if ($user->cannot('viewAny', Customer::class)) {
             return response()->json([
                 'res' => false,
                 'msg' => 'User is not authorized !',
                 'data' => ""
             ]);
         }
 
-        $response = $this->customerService->delete($customerKey);
+        $response = $this->customerService->delete($request->all());
```

Comment:
@arijit1111 I believe it should be the customer ID, which we are deleting from DB, instead of whole request object

Reply:
It is only passing customers id array and validation added in DeleteCustomerRequest